examples/create_attribute/create_attribute.py

```diff
@@ -10,7 +10,7 @@
 Full documentation is provided at http://python.dronekit.io/examples/create_attribute.html
 """
 
-from dronekit import connect
+from dronekit import connect, Vehicle
```

Comment:
Oh, this should change only to `from dronekit import ____`. Didn't notice the `.lib` part.